src/applications/disability-benefits/686c-674/config/chapters/veteran-information/veteran-address/veteran-address.js

```diff
@@ -7,22 +7,29 @@ export const schema = {
   required: [],
   properties: {
     veteranAddress: buildAddressSchema(true),
-    phoneNumber: genericSchemas.phoneInput,
```

Comment:
Instead of using `moreVeteranInformation`, would it be more accurate to wrap `veteranAddress`, `phoneNumber`, and `emailAddress` in a parent object called `veteranContactInformation`?

Reply:
Fixed this